crates/templates/src/run.rs

```diff
@@ -19,9 +19,6 @@ use crate::{
     template::Template,
 };
 
-/// A set of partials to be included in a Liquid template.
-type PartialsBuilder = liquid::partials::EagerCompiler<liquid::partials::InMemorySource>;
```

Comment:
Removing this is making the compile fail - I am not sure how you were even able to test it in the current state!  Whatever the intent, I suggest we back out the partials changes from this PR - I cannot imagine any reason they'd need to change for git init support.